src/flat-button.jsx

```diff
@@ -222,9 +283,9 @@ const FlatButton = React.createClass({
     this.props.onMouseLeave(e);
   },
 
-  _handleTouchStart(e) {
+  _handleTouchTap(e) {
     this.setState({touch: true});
```

Comment:
> I guess it makes sense to remove it since the only use it to pass it down to EnhancedButton

No, I think that it's useful to set the `touch` at the right moment. But I don't know when it's the "right" moment.